### PR TITLE
NP-118 added expand panel

### PR DESCRIPTION
**Pull request by Eiriknil**

### CODE REVIEW

File: src/pages/resource/PublicationPanel.tsx [hunk 1]
```diff
@@ -1,18 +1,80 @@
...
-    <div className="header">Velg publikasjon</div>
+    <div className="header">{t('Choose publication')}</div>
     <div className="panel-content">
       <div className="selector-wrapper">
-        <PublicationSourceSelector type="file"> </PublicationSourceSelector>
-        <PublicationSourceSelector type="link"> </PublicationSourceSelector>
-        <PublicationSourceSelector type="orcid"> </PublicationSourceSelector>
+        <ExpansionPanel className="publication-selector">
+          <ExpansionPanelSummary 
+              expandIcon={<ExpandMoreIcon />}
```
Comment (sarahse) [inline src/pages/resource/PublicationPanel.tsx]: Trekk ut i egen komponent
+ mer beskrivende aria-controls
+ hver expansionPanel må ha forskjellig id

File: src/pages/resource/LinkPublicationPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import PublicationExpansionPanel from './PublicationExpansionPanel';
+import { TextField, Button } from '@material-ui/core';
```
Comment (elkebab) [inline src/pages/resource/LinkPublicationPanel.tsx]: Material UI anbefaler path imports over named imports for å minimere build size: https://material-ui.com/guides/minimizing-bundle-size/#option-1

File: src/pages/resource/LoadPublicationPanel.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PublicationExpansionPanel from './PublicationExpansionPanel';
+
+import CloudDownloadIcon from '@material-ui/icons/CloudDownload';
+
+import { useTranslation } from 'react-i18next';
+
+interface LoadPanelProps {
```
Comment (sarahse) [inline src/pages/resource/LoadPublicationPanel.tsx]: LoadPublicationPanelProps **

File: src/pages/resource/LinkPublicationPanel.tsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import PublicationExpansionPanel from './PublicationExpansionPanel';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import LinkIcon from '@material-ui/icons/Link';
+
+import { useTranslation } from 'react-i18next';
+
+interface LinkPanelProps {
```
Comment (sarahse) [inline src/pages/resource/LinkPublicationPanel.tsx]: LinkPublicationPanelProps **

File: src/pages/resource/OrcidPublicationPanel.tsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PublicationExpansionPanel from './PublicationExpansionPanel';
+
+import SearchIcon from '@material-ui/icons/Search';
+
+import { useTranslation } from 'react-i18next';
+
+interface OrcidPanelProps {
```
Comment (sarahse) [inline src/pages/resource/OrcidPublicationPanel.tsx]: OrcidPublicationPanelProps **

File: src/pages/resource/PublicationExpansionPanel.tsx [hunk 6]
```diff
@@ -0,0 +1,39 @@
+import React, { ReactNode } from 'react';
+
+import ExpansionPanel from '@material-ui/core/ExpansionPanel';
+import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
+import ExpansionPanelDetails  from '@material-ui/core/ExpansionPanelDetails';
+
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+
+export interface PanelProps {
```
Comment (sarahse) [inline src/pages/resource/PublicationExpansionPanel.tsx]: PublicationExpansionPanelProps **

**Pull request merged**